src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/OSPlatform.cs

```diff
@@ -15,8 +17,11 @@ namespace System.Runtime.InteropServices
 
         public static OSPlatform Linux { get; } = new OSPlatform("LINUX");
 
+        [EditorBrowsable(EditorBrowsableState.Never)] // https://github.com/dotnet/runtime/issues/33331#issuecomment-650326500
         public static OSPlatform OSX { get; } = new OSPlatform("OSX");
 
+        public static OSPlatform macOS { get; } = new OSPlatform(nameof(macOS));
```

Comment:
nit: this should probably be `new OSPlatform(nameof("MACOS"))` to be consistent with the others (I know we're adding ignore-case comparison)